gellish_communicator/Create_output_file.py

```diff
@@ -90,44 +90,51 @@ def Open_output_file(expressions, subject_name, lang_name, serialization):
-                                               title="Enter a file name",
-                                               initialdir=ini_out_path,
-                                               initialfile=ini_file_name)
-    if output_file == '':
+##    title = serialization + ' files'
+##    extension = '*.' + serialization
+##    output_file = filedialog.asksaveasfilename(filetypes=((title, extension),
+##                                                          ("All files", "*.*")),
+##                                               title="Enter a file name",
+##                                               initialdir=ini_out_path,
+##                                               initialfile=ini_file_name)
+    output_file_name = ini_file_name
+    if output_file_name == '':
         output_file = 'Results.' + serialization
         if lang_name == 'Nederlands':
             print('***De filenaam voor opslaan is blanco of the file selectie is gecancelled. '
-                  'De file is opgeslagen met de naam ' + output_file)
+                  'De file met naam ' + output_file + ' is niet opgeslagen')
         else:
             print('***File name for saving is blank or file selection is cancelled. '
-                  'The file is saved under name ' + output_file)
-    Save_expressions_in_file(expressions, output_file, header1, serialization)
+                  'The file with name ' + output_file + 'is not saved')
+    else:
+        Save_expressions_in_file(expressions, output_file_name, header1, serialization)
+
+def fileupload_on_success(widget, filename):
+    print('File upload success: ' + filename)
 
+def fileupload_on_failed(widget, filename):
+    print('File upload failed: ' + filename)
 
 def Save_expressions_in_file(expressions, output_file, header1, serialization):
     '''Write expressions to an output file in an CSV or RDF serialization'''
 
     if serialization == 'csv':
         # Save the result_expr expressions in a CSV file, preceeded by three header lines.
         try:
-            f = open(output_file, mode='w', newline='', encoding='utf-8')
-            fileWriter = csv.writer(f, dialect='excel', delimiter=';')
+            f = open(output_file, mode='a', newline='', encoding='utf-8')
+            file_writer = csv.writer(f, dialect='excel', delimiter=';')
 
             # Write header rows and expressions
-            fileWriter.writerow(header1)
-            fileWriter.writerow(expr_col_ids)
-            fileWriter.writerow(header3)
+            file_writer.writerow(header1)
+            file_writer.writerow(expr_col_ids)
+            file_writer.writerow(header3)
             for expression in expressions:
-                fileWriter.writerow(expression)
+                file_writer.writerow(expression)
 
             f.close()
 
             # Open the written file in a CSV viewer (e.g. Excel)
-            open_file(output_file)
+            # open_file(output_file)
```

Comment:
If this was commented out, why not remove it altogether?

Reply:
It is intended to activate it again after development and testing is completed.